### ISSUE TITLE
cannot set title attribute on ProgressStep 

### ISSUE THREAD

**Issue by prateek04** [carbon-design-system/carbon#6292]
On Hover the progress step we should show the whole text.
So, we need to pass "title" attribute  as props 
for the same.

**Comment by prateek04** [comment]
@tw15egan I can see the issue has been closed. But can you tell me which version of carbon-components-react shall I install?
Below are the packages and respective versions I am using now
"carbon-components": "10.7.1",
  "carbon-components-react": "^7.13.0",

**Comment by tw15egan** [comment]
The fix has been merged in, but it has not been released yet. It will most likely be bundled in the next `10.15` release. Here is a link to our release radar which shows the cadence of our releases.
